### ISSUE TITLE
Indentation after importing operators

### ISSUE THREAD

**Issue by yuyichao** [JuliaLang/julia-emacs#12]
End of line operators doesn't always require indentation on the next line

```
module A
import Base: *
    a = 1
b = *
    c = 2
end
```

I guess this was introduced in https://github.com/JuliaLang/julia/pull/15156. Not sure this can be fixed without a full parser though...

**Comment by tpapp** [comment]
I have a fix for this, which I will submit once #37 is merged.

**Comment by non-Jedi** [comment]
@tpapp did you ever submit this fix?

**Comment by tpapp** [comment]
No. I have a branch which is severely outdated and provides a partial fix, if no one wants to take it up I will do it if #87 is merged.

**Comment by ronisbr** [comment]
Sorry about the dup.

To solve this, can’t we just check if the line stars with `import` here: https://github.com/JuliaEditorSupport/julia-emacs/blob/839726af374991223ffe56f7e7d10bef81033d09/julia-mode.el#L508

**Comment by yuyichao** [comment]
Note that the title is not completely accurate. It also applies to the `b = *` case, as shown in the example. Basically any time when an operator is used not as an operator.

**Comment by ronisbr** [comment]
Hum, I see. However, since the case:

```julia
import Base: *
```

should be way more common than the other, can we implement this temporary fix until we have a full parser?

**Comment by ronisbr** [comment]
I gave a try to solve it, but now I understand why we need a full parser. The indentation of lines after an `import` is treated here:

https://github.com/JuliaEditorSupport/julia-emacs/blob/839726af374991223ffe56f7e7d10bef81033d09/julia-mode.el#L543

instead of the previous function I marked. However, it does not seem easy to separate the code that mark a code line “unfinished” with that code that search for continuation in `import`. At least in my elisp level.

**Comment by ronisbr** [comment]
This issue is not present in tree-sitter implementation:

```julia
module A
import Base: *
a = 1
b = *
c = 2
end
```
